index.html

```diff
@@ -5,12 +5,94 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>ODIN-HTML-CSS</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
-    <script src="main.js"></script>
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
   </head>
   <body>
+    <div class="container" id="navigation">
+      <ul style="margin-right: 8px">
+        <li>
+          <p class="body-text">
+            Already have an Intuit account?
+          </p>
+        </li>
+        <li>
+          <p class="body-text body-text-link">
+            I forgot my User ID or Password
+          </p>
+        </li>
+      </ul>
+      <button id="signin-button">
+        <i class="fas fa-lock lock-icon"></i>Sing In
+      </button>
+    </div>
     <main>
-      <h1>HTML-CSS</h1>
+      <div class="container center">
+        <h1 id="logo">INTUIT</h1>
+      </div>
+      <div class="container col">
+        <div class="container center-vertical" style="padding: 16px">
+          <span class="container feature"
+            ><i class="fab fa-renren icon"></i>
+            <p class="body-text subheadertext">mint</p></span
+          >
+          <span class="container feature"
+            ><i class="fab fa-drupal icon "></i>
+            <p class="body-text subheadertext">quickbooks</p></span
+          >
+          <span class="container feature"
+            ><i class="fas fa-stamp icon"></i>
+            <p class="body-text subheadertext">turbotax</p></span
```

Comment:
Element p not allowed as child of element span in this context. (Suppressing further errors from this subtree.)
The element p cannot be used as the child element of span. Ensure this in all occurrences.